src/components/Popover/Popover.scss

```diff
@@ -42,8 +42,7 @@ $vertical-motion-offset: rem(-5px);
 }
 
 .fullWidth {
-  margin: 0;
-  margin-top: $visible-portion-of-arrow;
+  margin: $visible-portion-of-arrow auto 0;
```

Comment:
CSS shorthands are always harder to read than long form. Using the three property shorthand instead of 4 is especially difficult. Can you add the fourth value so it's less 🤯 ?